### PR TITLE
Enable ci-ingress-gce-downgrade-e2e

### PR DESCRIPTION
**Pull request by rramkumar1**
See title.

/assign @krzyzacy 
cc @MrHohn

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -8229,7 +8229,6 @@ periodics:
         secretName: ssh-key-secret
     - name: var-lib-docker
       emptyDir: {}
-  # TODO(rramkumar): Job does nothing yet.
 - name: ci-ingress-gce-downgrade-e2e
   agent: kubernetes
   interval: 24h
```
Comment (krzyzacy) [inline prow/config.yaml]: tigger it a bit more often?

**Comment by krzyzacy** [comment]
/lgtm
/hold
one comment, otherwise feel free to `/hold cancel` once you are happy about the interval

File: prow/config.yaml (hunk 1 above)
Comment (rramkumar1) [reply prow/config.yaml]: Done

**Pull request merged**